clientSide/save_cow/src/Components/NavBar/NavBar.css

```diff
@@ -0,0 +1,62 @@
+.nav-bar{
+    display: flex;
+    height: 50px;
+    width: 100%;
+    background-color: #1c49c2;
+}
+.profile-image{
+    height: 35px;
+    width: 35px;
+}
+.nav-bar > div{
+    color: white;
+}
+.GORAKSHA{
+    font-size: 20px;
+    margin: 13px 50px 10px 200px;
```

Comment:
every class have same properties please all those properties in one class and use that class name in your tags